### ISSUE TITLE
InputNumber: PageSpeed Accessibility -  Buttons do not have an accessible name

### ISSUE THREAD

**Issue by zawasp** [primefaces/primereact#5269]
### Describe the bug

Up/down buttons do not have an accessible name.

<img width="717" alt="image" src="https://github.com/primefaces/primereact/assets/2464830/0594b0f1-d003-4af0-8070-22aeae2da1d2">

### PrimeReact version

10.0.9

### React version

18.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Expected behavior

Up/down buttons should have an accessible name

**Comment by melloware** [comment]
I implemented it how PrimeVue handles it with `aria-hidden` since its not tabbable `tabindex=-1`
